front_end/src/components/BookComponents/Book.js

```diff
@@ -31,7 +32,7 @@ const Book = (props) => {
 		<tr>
 			<td className="book-list__th">{book.displayId}</td>
 			<td className="book-list__th">{book.name}</td>
-			<td className="book-list__th">{author.name}</td>
+			<td className="book-list__th">{book.author.name}</td>
```

Comment:
Don't use long object names:

###NOT TO DO
```suggestion
			<td className="book-list__th">{book.author.name}</td>
```

###DOING BETTER

`
```suggestion
                        const author = book.author;
                        
			<td className="book-list__th">{author}</td>
```